Cargo.toml

```diff
@@ -133,6 +133,11 @@ path = "src/bin/ipa_bench/ipa_bench.rs"
 required-features = ["cli", "enable-serde"]
 bench = false
 
+[[bin]]
+name = "report_collector"
+required-features = ["cli", "test-fixture", "web-app", "weak-field"]
```

Comment:
not sure if we want to expose `Fp31` to the clients, but I don't think we can fix it easily now. Maybe we can follow up with removing it from `test-fixture` and `rc` binary?

Reply:
Removed dependency on Fp31 atleast